docs/src/running-validator/validator-reqs.md

```diff
@@ -2,35 +2,31 @@
 title: Validator Requirements
 ---
 
-## Hardware
-
-- CPU Recommendations
-  - We recommend a CPU with the highest number of cores as possible. AMD Threadripper or Intel Server \(Xeon\) CPUs are fine.
-  - We recommend AMD Threadripper as you get a larger number of cores for parallelization compared to Intel.
-  - Threadripper also has a cost-per-core advantage and a greater number of PCIe lanes compared to the equivalent Intel part. PoH \(Proof of History\) is based on sha256 and Threadripper also supports sha256 hardware instructions.
-- SSD size and I/O style \(SATA vs NVMe/M.2\) for a validator
-  - Minimum example - Samsung 860 Evo 2TB
-  - Mid-range example - Samsung 860 Evo 4TB
-  - High-end example - Samsung 860 Evo 4TB
+## Hardware Recommendations
+
+- CPU
+  - 12 cores / 24 threads, or more
+  - 2.8GHz, or faster
+  - AVX2 instruction support
```

Comment:
If we want to be very generic, thread count is not really important SMT is generally 10-15% at best improvement. Clock speed may not be that important if IPC is high. AVX2 not required, just need to build from source.

Practically, these are good suggestions for most shipping x86 chips though.

Reply:
> If we want to be very generic, thread count is not really important SMT is generally 10-15% at best improvement.

I mainly mentioned threads here for the clarity that by cores, we mean physical cores.

> Clock speed may not be that important if IPC is high.

2.8GHz was chosen being what someone asking on Discord is likely to get

> AVX2 not required, just need to build from source.

I'll add a note that this is purely a requirement of using our release binaries
 
> Practically, these are good suggestions for most shipping x86 chips though.

:+1:  That's what I was going for!  The guiding principle was, "what have people been complaining about or championing on Discord for the last 2 mo" :joy: